src/python/pants/bin/local_pants_runner.py

```diff
@@ -140,10 +136,10 @@ def create(cls, exiter, args, env, target_roots=None, daemon_graph_session=None,
     options, build_config, options_bootstrapper = cls.parse_options(
       args,
       env,
-      setup_logging=True,
       options_bootstrapper=options_bootstrapper,
     )
     global_options = options.for_global_scope()
+    setup_logging_from_options(global_options)
```

Comment:
Might be worth noting that this change only works because of the `encapsulated_logger` in `DaemonPantsRunner`, and therefore we don't have to gate logging setup anymore.